lib/ansible/config/base.yml

```diff
@@ -1007,59 +1126,66 @@ DEFAULT_TEST_PLUGIN_PATH:
   type: pathlist
   yaml: {key: defaults.test_plugins}
 DEFAULT_TIMEOUT:
+  name: Connection timeout
   default: 10
-  description: 'TODO: write it'
+  description: This is the default timeout for connection plugins to use.
   env: [{name: ANSIBLE_TIMEOUT}]
   ini:
   - {key: timeout, section: defaults}
   type: integer
-  yaml: {key: defaults.timeout}
 DEFAULT_TRANSPORT:
+  name: Connection plugin
   default: smart
   description: "Default connection plugin to use, the 'smart' option will toggle between 'ssh' and 'paramiko' depending on controller OS and ssh versions"
   env: [{name: ANSIBLE_TRANSPORT}]
   ini:
   - {key: transport, section: defaults}
-  yaml: {key: defaults.transport}
 DEFAULT_UNDEFINED_VAR_BEHAVIOR:
+  name: Jinja2 fail on undefined
   default: True
-  description: 'TODO: write it'
+  version_added: "1.3"
+  description:
+    - When True, this causes ansible templating to fail steps that reference variable names that are likely typoed.
+    - Otherwise, any ‘{{ template_expression }}’ that contains undefined variables will be rendered in a template or ansible action line exactly as written.
   env: [{name: ANSIBLE_ERROR_ON_UNDEFINED_VARS}]
   ini:
   - {key: error_on_undefined_vars, section: defaults}
   type: boolean
-  yaml: {key: defaults.error_on_undefined_vars}
 DEFAULT_VARS_PLUGIN_PATH:
+  name: Vars plugins path
```

Comment:
Other plugin path config names[1] capitalize 'Plugins' and/or 'Path'. plugin path names need to be updated so they are consistent.

'Action plugins path' -> 'Action Plugins Path'
'Vars plugins path' -> 'Vars Plugins Path'

[1] ie, 'Callback Plugins Path',
``` 
['Action plugins path',
 'Cache Plugins Path',
 'Callback Plugins Path',
 'Callback Plugins Path',
 'Inventory Plugins Path',
 'Jinja2 Filter Plugins Path',
 'Jinja2 Test Plugins Path',
 'Lookup Plugins Path',
 'Strategy Plugins Path',
 'Vars plugins path']
```